piksi_tools/pickle_to_json.py

```diff
@@ -0,0 +1,64 @@
+
+from sbp.client.loggers.json_logger import JSONLogIterator
+from sbp.client.loggers.pickle_logger import PickleLogIterator
+import json
+import struct
+import sbp
+import sbp.table
+import sys
+sys.modules['sbp_piksi'] = sbp  # creates a packageA entry in sys.modules
+
+def rewrite(datafiles, process_functions=[]):
+  """
+  Rewrite pickle file as a json file with a similar name
+  Parameters
+  ----------
+  datafiles : list
+      list of files on which to operate.
+  process_functions : list
+      list of functions to call on the message as it is converted
+  """
+  for f in datafiles:
+      with open(f + ".json", 'w+') as log_file:
+        with PickleLogIterator(f) as log:
+          for delta, timestamp, msg in log.next():
+            for process in process_functions:
+                msg = process(msg)
+            m = struct.pack("<HHB", msg.msg_type, msg.sender, len(msg.payload))
+            m += msg.payload
+            crc = sbp.crc16(m[1:])
+            msg.crc = crc
+            m = {"delta": delta,
+                 "timestamp": timestamp,
+                 "data": sbp.table.dispatch(msg).to_json_dict()}
+            log_file.write(json.dumps(m) + "\n")
+          print "Succesfully wrote json output to {0}".format(log_file)
+
+def translate_emphemeris(msg):
```

Comment:
nit: ephemeris is misspelled. 

Backwards compatibility is a hairy question, but this is fine for time being. I might generalize the application of `translate_ephemeris` to `migrate_msg` or something, and provide a list of rewrites that can be applied. This isn't the last case we're going to have something like this.